backend/app/assets/stylesheets/spree/backend/sections/_orders.scss

```diff
@@ -1,18 +1,3 @@
-.line-item.editing {
-  .edit-line-item,
-  .delete-line-item,
-  .line-item-qty-show {
-    display: none !important;
-  }
-}
-
-.line-item:not(.editing) {
-  .save-line-item,
-  .cancel-line-item,
-  .line-item-qty-edit {
```

Comment:
`.line-item-qty-edit` and `.line-item-qty-show` are JS utility classes used in the old JS only AFAIK. Maybe there are others. Not mandatory, for sure. But I like to clean up as we have the chance.